<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">padding: 3px</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: white; border-color: black; border-width: 3px; border-style: solid; padding: 3px</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>17</number>
    </property>
    <property name="rightMargin">
     <number>17</number>
    </property>
    <item row="2" column="0">
     <widget class="QLabel" name="selectLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 0px</string>
      </property>
      <property name="text">
       <string>Select the required from the available element types for ICP below:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="welcomeLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20px; background-color: rgb(16, 72, 145); color: white; border-radius: 5px; margin: 7px; border-width: 1px</string>
      </property>
      <property name="text">
       <string>SQL Queries Generator</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; padding-left: 5px; padding-right: 5px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="instanceLabel">
          <property name="styleSheet">
           <string notr="true">font: 12px;
font-weight: bold;
padding: 5px 0px 5px 5px;
border-width: 0px</string>
          </property>
          <property name="text">
           <string>Instance URI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="instanceLineEdit">
          <property name="styleSheet">
           <string notr="true">border-width: 1px; padding-left: 5px; padding-right: 5px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="usernameLabel">
          <property name="styleSheet">
           <string notr="true">font: 12px;
font-weight: bold;
padding: 5px;
border-width: 0px</string>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameLineEdit">
          <property name="styleSheet">
           <string notr="true">border-width: 1px</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="styleSheet">
           <string notr="true">font: 12px;
font-weight: bold;
border-width: 0px</string>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-width: 1px; padding-left: 5px; padding-right: 5px;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="initiateButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
font: 12px;
font-weight: bold;
border-style: solid;
border-width: 2px;
border-color: black;
border-radius: 3px;
padding: 8px;
width: 55px}
QPushButton:hover{
background-color: rgb(0,0,128);
color:white;
font: 12px;
font-weight: bold;
border-style: solid;
border-width: 2px;
border-color: black;
border-radius: 3px;
padding: 8px;
width: 55px}</string>
          </property>
          <property name="text">
           <string>Initiate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
font: 12px;
font-weight: bold;
border-style: solid;
border-width: 2px;
border-color: black;
border-radius: 3px;
padding: 8px;
width: 55px}
QPushButton:hover{
background-color: rgb(0,0,128);
color:white;
font: 12px;
font-weight: bold;
border-style: solid;
border-width: 2px;
border-color: black;
border-radius: 3px;
padding: 8px;
width: 55px}</string>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
font: 12px;
font-weight: bold;
border-style: solid;
border-width: 2px;
border-color: black;
border-radius: 3px;
padding: 8px;
width: 55px;
margin-bottom: 8px}
QPushButton:hover{
background-color: rgb(0,0,128);
color: white;
font: 12px;
font-weight: bold;
border-style: solid;
border-width: 2px;
border-color: black;
border-radius: 3px;
padding: 8px;
width: 55px;
margin-bottom: 8px}
</string>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="searchLineEdit">
      <property name="styleSheet">
       <string notr="true">border-width: 1px</string>
      </property>
      <property name="placeholderText">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
